packages/hash/frontend/src/pages/[account-slug]/[page-slug].page.tsx

```diff
@@ -239,6 +236,25 @@ const Page: NextPageWithLayout<PageProps> = ({ blocks }) => {
     pageHeaderRef.current.scrollIntoView({ behavior: "smooth" });
   };
 
+  const { data: pageComments } = usePageComments(pageEntityId);
+
+  const [Container, paddingLeftFormula, paddingRightFormula] = useMemo(() => {
+    const commentsContainerWidth = pageComments?.length ? COMMENTS_WIDTH : 0;
+
+    const paddingLeft = `max(calc((100% - ${
+      PAGE_CONTENT_WIDTH + commentsContainerWidth + PAGE_MIN_PADDING
+    }px) / 2), ${PAGE_MIN_PADDING}px)`;
+    const paddingRight = `calc((100% - ${PAGE_CONTENT_WIDTH}px - ${paddingLeft}))`;
+
+    return [
+      styled("div")({
+        padding: `${PAGE_MIN_PADDING}px ${paddingRight} 0 ${paddingLeft} `,
+      }),
```

Comment:
This is *very* strange – why not have this come as a value out of the memo and pass it in to a `Box` via `sx`?

Reply:
You should avoid dynamically creating a component like this whereever possible. Just make a wrapper component if there's code you need to reuse.